Install.sh

```diff
@@ -1,44 +1,36 @@
 #!/bin/bash
-# TODO: update readme.
+
+# Source logo and usage.
+source ./lib/utils.sh
+
+# Move other files in this dir to /home/$USER/filefighter/
+SCRIPTS_LOCATION="/home/$USER/filefighter"
```

Comment:
can you check if that folder exists? Otherwise create it or place files in /home/filefighter

Reply:
yes.

Reply:
`Home directory for the current user not found. If you are root, switch to a user with a directory under /home/`